src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -117,14 +118,14 @@ void fillInnerParts() {
         contactListPanel = new ContactListPanel(logic.getFilteredContactList());
         contactListPanelPlaceholder.getChildren().add(contactListPanel.getRoot());
 
-        meetingListPanel = new MeetingListPanel(logic.getFilteredMeetingList());
-        meetingListPanelPlaceholder.getChildren().add(meetingListPanel.getRoot());
+        //meetingListPanel = new MeetingListPanel(logic.getFilteredMeetingList());
```

Comment:
Maybe these lines can be deleted rather than commented out?

Reply:
Will do so in subsequent PR's affecting meeting list.